dials.go

```diff
@@ -24,6 +24,15 @@ type Params struct {
 	//  - a Verify() method fails after re-stacking when a new version is
 	//    provided by a watching source
 	OnWatchedError WatchedErrorHandler
+
+	// SkipInitialVerification skips the initial call to `Verify()` on any
+	// configurations that implement the `VerifiedConfig` interface.  In cases
+	// where Watching sources are depended upon to provide values that will
+	// successfully verify on the very first access, one should set this to
+	// `true`.  `sourcewrap.Blank` is one such example where values are not
+	// initially available and would not be able to be considered for initial
+	// verification.
```

Comment:
Hmm, maybe we should move this portion into the type-level docstring for `Blank` itself.

Reply:
How about "belt and suspenders?"  I've added a section the type description for `Blank` calling it out as well.